### PR TITLE
Update LocalSettings.php to change external link target

### PR DESCRIPTION
**Pull request by TheDavidJohnson**
Intended to address #10 (would require testing within Sandstorm environment)

Adds MediaWiki setting to cause external links to open in new tab. Reference: https://www.mediawiki.org/wiki/Manual:$wgExternalLinkTarget

### CODE REVIEW

**Comment by jparyani** [comment]
Thanks!

**Comment by TheDavidJohnson** [comment]
Excellent! And hey... since I'm brand new to Sandstorm... will this change trickle down to my Sandstorm install? Or should I do something manually to acquire the latest version?  (Sorry if this is the wrong place for this question!)

**Comment by jparyani** [comment]
So, I'm currently building the new spk (that's the package format for Sandstorm apps). I'll upload it to the app store (https://apps.sandstorm.io/) within the hour and after it's approved (usually takes only a few minutes) it will be available for install there.

Also, all Sandstorm servers will check for app updates nightly (unless it's disabled by your admin) and prompt the user with a notification asking if they wish to update.

So in summary, either go to https://apps.sandstorm.io/app/y88wavuqwz0p3tjcqtdt8egauq9hpnzr1s9efq1d63rwtj1w0ech in an hour or 2 and manually update, or wait for the autoupdate notification tomorrow.

**Comment by TheDavidJohnson** [comment]
Awesome... thanks!

**Comment by TheDavidJohnson** [comment]
Just confirming that this works on my instance of MediaWiki on Sandstorm. After updating, links are indeed opening in a new browser tab. Thanks @jparyani!
